packages/wallets/.stylelintrc.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
                 message: 'Expected selector to match BEM CSS pattern and to be prefixed with "wallets-"',
             },
         ],
+        'unit-allowed-list': ['%', 'deg', 'em', 'fr', 'ms', 'rem', 's', 'svh', 'vh', 'vw'],
```

Comment:
is there a way to only exclude `px` rather than list one by one units?

Reply:
I think its better to have predefined units like this. So we know what we using in our project